packages/front-end/components/Features/FeaturesOverview.tsx

```diff
@@ -113,6 +116,12 @@ export default function FeaturesOverview({
   const router = useRouter();
   const { fid } = router.query;
 
+  const { data } = useApi<{
+    archetype: ArchetypeInterface[];
+  }>("/archetype");
+
+  const archetypes = data?.archetype || [];
```

Comment:
This will cause flashing on the page.  Even when you do have archetypes, this will initially be an empty array on first load, so the banner will show for a second until this api call finishes.